llvm/lib/Target/AMDGPU/AsmParser/AMDGPUAsmParser.cpp

```diff
@@ -9748,6 +9759,20 @@ ParseStatus AMDGPUAsmParser::parseEndpgm(OperandVector &Operands) {
 
 bool AMDGPUOperand::isEndpgm() const { return isImmTy(ImmTyEndpgm); }
 
+//===----------------------------------------------------------------------===//
+// BITOP3
+//===----------------------------------------------------------------------===//
+
+ParseStatus AMDGPUAsmParser::parseBitOp3(OperandVector &Operands) {
+  ParseStatus Res =
+      parseIntWithPrefix("bitop3", Operands, AMDGPUOperand::ImmTyBitOp3);
+  return Res;
+}
+
+AMDGPUOperand::Ptr AMDGPUAsmParser::defaultBitOp3() const {
```

Comment:
Hi,

This method seems to be unused on main. WIll it be used in some coming patch or should it be removed?
gcc warns like
```
../lib/Target/AMDGPU/AsmParser/AMDGPUAsmParser.cpp:9809: warning: '{anonymous}::AMDGPUOperand::Ptr {anonymous}::AMDGPUAsmParser::defaultBitOp3() const' defined but not used [-Wunused-function]
 9809 | AMDGPUOperand::Ptr AMDGPUAsmParser::defaultBitOp3() const {
      | 
```

Reply:
#118648